packages/kit/src/core/build/index.js

```diff
@@ -1,14 +1,18 @@
 import fs from 'fs';
 import path from 'path';
-import { rimraf } from '../../utils/filesystem.js';
-import create_manifest_data from '../../core/create_manifest_data/index.js';
-import { copy_assets, posixify, resolve_entry } from '../utils.js';
-import { deep_merge, print_config_conflicts } from '../config/index.js';
-import { create_app } from '../../core/create_app/index.js';
-import vite from 'vite';
+
 import { svelte } from '@sveltejs/vite-plugin-svelte';
 import glob from 'tiny-glob/sync.js';
+import vite from 'vite';
+
+import { rimraf } from '../../utils/filesystem.js';
+import { deep_merge } from '../../utils/object.js';
+
```

Comment:
Is there a reason for the empty lines?

Reply:
Just like needing to group codes into related sectioned blocks, I find myself having a hard time reading and separating what imports a certain file relies on when it's jumbled up together, and giving these imports some "breathing space" makes the separation clearer between the used member syntax, especially when it's over 5-6 imports.